xchange-bitfinex/src/main/java/org/knowm/xchange/bitfinex/v2/dto/trade/Trade.java

```diff
@@ -37,7 +36,91 @@
   /** Fee currency */
   private String feeCurrency;
 
-  public Date getTimestamp() {
-    return new Date(timestamp);
+  public String getId() {
+    return id;
+  }
```

Comment:
this class uses lombok and getters and setters without any peculiar behaviour are not needed here

Reply:
Oh, I didn't see that: It is fixed now